### PR TITLE
traduccionn to Spanish

### PR DESCRIPTION
**Pull request by yabellini**
## Author: 

-  yabellini

## Language: 

-  Spanish

## Terms defined:

- epoch
- learning rate
-

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -10404,6 +10404,12 @@
       In [deep learning](#deep_learning), an epoch is one cycle in the deep learning process where all
       the training data has been fed to the algorithm once. Training a deep neural networks usually
       consists of multiple epochs.
+  es:
+    term: epoch (deep learning)
+    def: > 
+      En [deep learning](#deep_learning), un epoch es un ciclo completo en el proceso del aprendizaje profundo
```
Comment (nicoguaro) [inline glossary.yml:10410]: Here we are using "deep learning" and "aprendizaje profundo" at the same time. Should we keep just "aprendizaje profundo"?

**Review by nicoguaro — CHANGES_REQUESTED** [review]
I think that we should use "aprendizaje profundo" instead of "deep learning" in Spanish

File: glossary.yml [hunk 2]
```diff
@@ -10404,6 +10404,12 @@
       In [deep learning](#deep_learning), an epoch is one cycle in the deep learning process where all
       the training data has been fed to the algorithm once. Training a deep neural networks usually
       consists of multiple epochs.
+  es:
+    term: epoch (deep learning)
```
Comment (yabellini) [inline glossary.yml:10408]:
```suggestion
    term: epoch (aprendizaje profundo)
```

File: glossary.yml (hunk 1 above)
Comment (yabellini) [inline glossary.yml:10410]:
```suggestion
      En [aprendizaje profundo](#deep_learning), un epoch es un ciclo completo en el proceso del aprendizaje profundo 
```

File: glossary.yml [hunk 3]
```diff
@@ -10420,6 +10426,14 @@
       A large learning rate can speed up training, but the network might overshoot and miss the 
       minimum. A small learning rate will overshoot less, but will be slower. It can also get more 
       easily stuck in local minima.
+  es:
+    term: tasa de aprendizaje (deep learning)
```
Comment (yabellini) [inline glossary.yml:10430]:
```suggestion
    term: tasa de aprendizaje (aprendizaje profundo)
```

**Review by yabellini — COMMENTED** [review]
Cambios hechos de acuerdo al pedido de la revision.  Gracias @nicoguaro !

**Comment by elletjies** [comment]
@nicoguaro Could you check the changes to the translation again please?

**Review by nicoguaro — APPROVED** [review]

**Comment by elletjies** [comment]
@nicoguaro thank you for the review!

**Pull request merged**